brigadier/accounts/tests.py

```diff
@@ -24,15 +26,17 @@ def test_create_user_public_group_exists(self):
 
         """
         group_public = Group.objects.get(name='public')
-        response = self.client.post(
-            reverse('accounts:registration'),
-            {
-                'username': 'User_name',
-                'email': 'user@example.com',
-                'password1': '1234',
-                'password2': '1234',
-            }
-        )
+
+        with mock.patch('worker.email.tasks.send_verification_mail.delay') as m:
+            response = self.client.post(
+                reverse('accounts:registration'),
+                {
+                    'username': 'User_name',
+                    'email': 'user@example.com',
+                    'password1': '1234',
+                    'password2': '1234',
+                }
+            )
         self.assertEqual(response.status_code, 302)
         self.assertEqual(response.url, reverse('accounts:registration_done'))
```

Comment:
you've created a mock for `worker.email.tasks.send_verification_mail.delay` but you don't check was it called during account registration